docs/source/tutorials/configuration.ipynb

```diff
@@ -1,20 +1,452 @@
 {
```

Comment:
<p>Maybe we can combine all 4 code boxes into a one.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2876/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>